python/src/viewer.py

```diff
@@ -844,18 +844,18 @@ def PlotDesign(
     ----------
     design : 2-d sequence of float
         The sample.
-    figure : a Matplotlib figure.
+    figure : a Matplotlib figure, optional
         If this is not None, then create a new figure.
         Otherwise, use the existing figure.
-    axes : a Matplotlib axis.
+    axes : a Matplotlib axis, optional
         If empty, then create new axes.
-    bounds: :class:`~openturns.Interval`
+    bounds : :class:`~openturns.Interval`, optional
         Bounds of the plot. By default, compute the bounds from the sample.
-    subdivisions : a list of integers
+    subdivisions : a list of integers, optional
         Number of subdivisions in the each direction.
         By default, set the number of subdivisions in each direction
         as equal to the sample size.
```

Comment:
This seems to be a poor choice. Couldn't we set it to something like `[5] * dim` ?

```python
fig = viewer.PlotDesign(ot.Normal(2).getSample(100), subdivisions=[5,5])
```
![image](https://github.com/openturns/openturns/assets/31989332/dc32ae7e-ce6a-4b26-8584-9ad649c78b93)

Whereas with the default:

```python
fig = viewer.PlotDesign(ot.Normal(2).getSample(100))
```

we get something like:

![image](https://github.com/openturns/openturns/assets/31989332/b4e2409d-7a74-4b0e-a5f3-876d42a36907)

Reply:
Also, the boundaries should not fit the data exactly but leave a small margin.

Reply:
Done: #2540